src/code/z_collision_check.c

```diff
@@ -2666,8 +2666,8 @@ void CollisionCheck_SetOCvsOC(Collider* left, ColliderInfo* leftInfo, Vec3f* lef
     if (leftActor == NULL || rightActor == NULL || left->ocFlags1 & OC1_NO_PUSH || right->ocFlags1 & OC1_NO_PUSH) {
         return;
     }
-    rightMassType = CollisionCheck_GetMassType(leftActor->colChkInfo.mass);
-    leftMassType = CollisionCheck_GetMassType(rightActor->colChkInfo.mass);
+    leftMassType = CollisionCheck_GetMassType(leftActor->colChkInfo.mass);
+    rightMassType = CollisionCheck_GetMassType(rightActor->colChkInfo.mass);
```

Comment:
unrelated to the actual fix, ive always found the usage of "left" and "right" here kinda weird. I guess something like 1 and 2 isnt that much better, but implying some kind of direction with them has always sounded weird to me

dont have to do anything about it in this pr, but just putting that thought out there